modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -316,8 +316,6 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ]);
 
       if ($next_step_id == 'complete') {
-        // @todo Do this earlier in the process.
-        $this->order->cart = FALSE;
```

Comment:
Removed this, since the `place` transition will fire `commerce_cart` to do `finalizeCart`